build_doc.sh

```diff
@@ -50,10 +50,21 @@ function clean_docs() {
 
 function build_html_docs() {
     pip install -e .[dev]
+    cp versioneer.py docs/.
```

Comment:
do we need this? I see you change the versioneer.py 
maybe @IsaacYangSLA can help comment on that